src/integer_mod_q/mat_zq/get.rs

```diff
@@ -26,139 +27,43 @@ use flint_sys::{
 use std::{fmt::Display, mem::MaybeUninit};
 
 impl MatZq {
-    /// Returns the modulus of the matrix as a [`Modulus`].
+    /// Create a [`MatZ`] from a [`MatZq`].
```

Comment:
Create might be a bit confusing. Maybe more: Creates a MatZ where each entry is the representative of the equivalence class of each entry.

or similar